packages/drivers/odsp-driver/src/fetchWithEpochValidation.ts

```diff
@@ -0,0 +1,125 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "assert";
+import { ITelemetryLogger } from "@fluidframework/common-definitions";
+import { createOdspNetworkError, fluidEpochMismatchError, OdspErrorType } from "@fluidframework/odsp-doclib-utils";
+import { fetchAndParseAsJSONHelper, fetchHelper, IOdspResponse } from "./odspUtils";
+import { ICacheEntry, IPersistedCache, IPersistedCacheValue } from "./odspCache";
+
+/**
+ * This class is a wrapper around fetch calls. It adds epoch to the request made so that the
+ * server can match it with its epoch value in order to match the version.
+ * It also validates the epoch value received in response of fetch calls. If the epoch does not match,
+ * then it also clears all the cached entries for the given container.
+ */
+export class FetchWithEpochValidation {
```

Comment:
EpochTracker seems much more concise name without loss of intention. "fetch" verb is present in methods.

Reply:
Renamed.